.github/workflows/pr_checks.yml

```diff
@@ -3,26 +3,36 @@ on:
   pull_request:
 
 jobs:
-  backend-unit-tests:
-    name: Backend Unit Tests
+  tests:
+    name: Containerized Tests
     runs-on: self-hosted
     env:
-      BUILD_DIR: tmp/pr_builds/PR_${{ github.event.number }}/backend_unit_tests
+      BUILD_DIR: tmp/pr_builds/PR_${{ github.event.number }}/
       DOCKER_PROJECT_NAME: backend-unit-tests-${{ github.event.number }}-${{ github.run_id }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
         with:
             path: ${{ env.BUILD_DIR }}
 
-      - name: Run Backend Unit Tests with Docker
+      - name: Backend Unit Tests with Docker
         working-directory: ${{ env.BUILD_DIR }}
-        run: docker compose -p ${{ env.DOCKER_PROJECT_NAME }} -f compose.testing.yml run --build backend-unit-tests
+        run: make backend_tests PROJECT_NAME=${{ env.DOCKER_PROJECT_NAME }}
 
-      - name: Cleanup Docker Resources
+      - name: Playwright E2E Tests with Docker
+        working-directory: ${{ env.BUILD_DIR }}
+        run: make e2e_tests PROJECT_NAME=${{ env.DOCKER_PROJECT_NAME }}
+
+      - name: Cleanup Docker Resources - Docker Compose Down
         if: always()
         working-directory: ${{ env.BUILD_DIR }}
-        run: |
-          docker compose -p ${{ env.DOCKER_PROJECT_NAME }} -f compose.testing.yml down --volumes
-          docker images --filter "label=com.docker.compose.project=${{ env.DOCKER_PROJECT_NAME }}" -q | xargs -r docker rmi -f
```

Comment:
The `|` operator preserves newlines which was causing issues, I tried using `>` which replaces newlines with spaces but then we would need `&&` or some other concat operator.

Decided to just split each into it's own step.

Reply:
Yeah you either need to do:
```
run: |
  command 1 \
  command 2
```
or:
```
run: >
  command 1 &&
  command 2
```